Solutions/NozomiNetworks/Data Connectors/Connector_CEF_NozomiNetworks.json

```diff
@@ -0,0 +1,121 @@
+{
+    "id": "NozomiNetworks",
+    "title": "Nozomi Networks",
+    "publisher": "Nozomi Networks",
+    "descriptionMarkdown": "The [Nozomi Networks](https://www.nozominetworks.com/) data connector provides the capability to ingest Nozomi Networks Events into Azure Sentinel. Refer to the Nozomi Networks PDF documentation for more information.",
+    "additionalRequirementBanner": "This data connector depends on a parser based on Kusto Function to work as expected [**NozomiNetworksEvents**](https://aka.ms/sentinel-NozomiNetworks-parser) which is deployed with the Azure Sentinel Solution.",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "Nozomi Networks",
+            "baseQuery": "NozomiNetworksEvents"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "Top 10 Devices", 
+            "query": "NozomiNetworksEvents\n | summarize count() by DvcHostname\n | top 10 by count_"
+        }
+    ],
+    "dataTypes": [
+        {
+            "name": "CommonSecurityLog (Nozomi Networks)",
```

Comment:
Remove the space Between Nozomi and Networks.

Reply:
done